src/api/app/controllers/webui/request_controller.rb

```diff
@@ -230,7 +230,6 @@ def add_role_request_dialog
   end
 
   def add_role_request
-    required_parameters :project, :role
```

Comment:
What about the routes?

Reply:
Param `project` is now added to the route.

Param `role` is not added, as it comes from a select box. In case it is not passed as parameter the controller will throw an exception.